cz-config-EXAMPLE.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
     { value: 'WIP', name: 'WIP:      Work in progress' },
   ],
 
+  additionalQuestions: [
```

Comment:
Thanks for the PR!

I believe the "standard" questions are suitable for most users.

Most people are going to copy and paste this example file. My only concern is they would be overwhelmed by the additional questions. 

The README file explains how to use the new field `additionalQuestions` if they need it. 

So it would LGTM if we leave this file `cz-config-EXAMPLE.js` untouched like a bare minimum to get started. Does it make sense?

Reply:
@leonardoanalista , Got it, readme is sufficient enough to explain `additionalQuestions` options.
Removed `additionalQuestions` from this file now

Reply:
Thank you @parveen14 and @samarpan-b!